GUI/ManageKspInstances.cs

```diff
@@ -127,25 +135,52 @@ private void UseSelectedInstance()
 
             try
             {
-                if (SetAsDefaultCheckbox.Checked)
-                {
-                    _manager.SetAutoStart(instName);
-                }
-
                 _manager.SetCurrentInstance(instName);
                 DialogResult = DialogResult.OK;
                 Close();
             }
             catch (NotKSPDirKraken k)
             {
-                GUI.user.RaiseError(Properties.Resources.ManageKspInstancesNotValid,
-                    new object[] { k.path });
+                GUI.user.RaiseError(Properties.Resources.ManageKspInstancesNotValid, k.path);
+            }
+        }
+
+        private void SetAsDefaultCheckbox_Click(object sender, EventArgs e)
```

Comment:
Could we react to `_CheckboxChanged` instead of `_Click`? The checkbox already has its own built in logic to change the value on clicking, and it's weird that we want to do the same in our code as well.

Reply:
I wanted to do that too, but it seems that this line
```csharp
SetAsDefaultCheckbox.Checked = _manager.AutoStartInstance?.Equals(instName) ?? false;
```
of `KSPInstancesListView_SelectedIndexChanged` triggers it too.

Reply:
Looking at it again, I'm not sure whether it's really that line that's causing trouble, but nevertheless the checkbox doesn't work anymore if I change it to react to `CheckedChanged` :/

Reply:
OK, I'm fine with this unless/until there's a problem...